File: Modul-5/Soal 2/tempCodeRunnerFile.py
def fathi(bila):
    return -bila if bila < 0 else bila
def ega(rihan, rey):
    return fathi(rihan-rey)
def main():
    a, c, b, d = map(int, input().split())
    ipk = ega(a,b)+ega(c,d)
    print(fathi(ipk))
main()

File: Modul-5/Soal 2/tempCodeRunnerFile.c
#include <stdio.h>
#include <math.h>
int fathi(int bila){return (bila<0)?-bila:bila;}
int ega(int rihan, int raka){return fathi(rihan-raka);}
int main () {
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &c, &b, &d);
    int ipk = ega(a,b)+ega(c,d);
    printf("%d", fathi(ipk));
}